src/main/java/net/sf/jabref/gui/ClipBoardManager.java

```diff
@@ -66,6 +66,7 @@ public String getClipboardContents() {
         return result;
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Can this be moved to a place where the unchecked conversion really occurs? If not, please add a comment why the suppression is needed at the whole method.